src/control/run_go_tests.sh

```diff
@@ -2,6 +2,59 @@
 ## Run linters across control plane code and execute Go tests
 set -e
 
+function find_build_source()
+{
+	BASE=$PWD
+	while true
+	do
+		path=$(realpath "${BASE}")
+		if [ "${path}" == "/" ]; then
+			break
+		fi
+		test -e "${path}/.build_vars.sh" && echo "${path}/.build_vars.sh" && return
+		BASE=$(dirname "${path}")
+	done
+	echo ""
+}
+
+function check_environment()
+{
+	if [ -z "$LD_LIBRARY_PATH" ]; then
+		echo "false" && return
+	fi
+
+	if [ -z "$CGO_LDFLAGS" ]; then
+		echo "false" && return
+	fi
+
+	if [ -z "$CGO_CFLAGS" ]; then
+		echo "false" && return
+	fi
+	echo "true" && return
+}
+
+function setup_environment()
+{
+	build_source=$(find_build_source)
+
+	if [ "${build_source}" == "" ]; then
+		echo "Unable to find .build_source.sh" && exit 1
+	fi
+
+	source "${build_source}"
+
+	LD_LIBRARY_PATH="${SL_PREFIX}/lib:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
```

Comment:
(style)  line over 80 characters

Reply:
Not reasonable to split before 80 characters.